packages/server/postgres/queries/getPollsByDiscussionId.ts

```diff
@@ -0,0 +1,7 @@
+import {getPollsByDiscussionIdQuery} from './generated/getPollsByDiscussionIdQuery'
```

Comment:
Rename this to `getPollsByDiscussionId`**`s`** so it's clear that it expects an array of discussionIds